itkwidgets/trait_types.py

```diff
@@ -183,7 +182,7 @@ def itkimage_from_json(js, manager=None):
     else:
         ImageType, dtype = _type_to_image(js['imageType'])
         decompressor = zstd.ZstdDecompressor()
-        pixelBufferArrayCompressed = np.frombuffer(base64.b64decode(js['compressedBase64Data']), dtype=dtype)
+        pixelBufferArrayCompressed = np.array(js['compressedData'], dtype=dtype)
```

Comment:
Maybe you should still use `np.frombuffer`? I think that way you avoid copying the memory? This is just off the top of my head, so you should probably double check though.

Reply:
`np.frombuffer` throws an error about missing `__buffer__`.

Any other suggestions welcome...

Reply:
Is this with python 2? I'm using this code somewhere else:
```python
if six.PY2:
    # Numpy does not support frombuffer with memoryview
    # on Python 2:
    buffer = value['buffer'].tobytes()
    n = np.frombuffer(buffer, dtype=value['dtype'])
else:
    n = np.frombuffer(value['buffer'], dtype=value['dtype'])
```

Reply:
(PS: I updated the code in the previous comment to use the same variable names as your code.)

Reply:
Excellent, thanks for the suggestion, @vidartf ! Adding in a follow-up commit...